nessie/src/test/java/org/apache/iceberg/nessie/TestNessieTable.java

```diff
@@ -92,15 +90,15 @@ public TestNessieTable() {
   @BeforeEach
   public void beforeEach(@NessieUri URI uri) throws IOException {
     super.beforeEach(uri);
-    this.tableLocation = new Path(catalog.createTable(TABLE_IDENTIFIER, schema).location());
+    this.tableLocation = new File(catalog.createTable(TABLE_IDENTIFIER, schema).location());
```

Comment:
I don't really see why this isn't a `String` that gets converted to `File` when needed.

Reply:
Hmm, I just followed the original author's style instead of refactoring it. 

Fixed now.